src/EditorFeatures/Test2/FindReferences/FindReferencesTests.OrdinaryMethodSymbols.vb

```diff
@@ -3254,6 +3254,28 @@ End Class
         }
         </Document>
     </Project>
+</Workspace>
+            Await TestAPIAndFeature(input, kind, host)
+        End Function
+        <WpfTheory, CombinatorialData, Trait(Traits.Feature, Traits.Features.FindReferences)>
+        Public Async Function TestOrdinaryMethodWithMissingReferences(kind As TestKind, host As TestHost) As Task
+            Dim input =
+<Workspace>
+    <Project Language="C#" CommonReferences="false">
+        <Document>
+        class C
+        {
+            // string will be an error type because we have no actual references.
+            private void {|Definition:Goo|}(string s) { }
+
+            void Bar()
+            {
+                [|Go$$o|]("");
+                [|Goo|](s);
+            }
+        }
+        </Document>
```

Comment:
have verified this test fails in OOP scenarios prior to the fix and passes post the fix.